### DISCUSSION TITLE
Not all reservation is not showing in API call

### DISCUSSION THREAD

**Discussion by Veljen** [Azure/ipam#255] [General]
we have deployment IPAM using app service and registered with app registration.

we use terraform azureipam_reservation module to reserve the block and this pipeline was run using SP, when we try to import the resource using user creds/token for the api is not showing the reserveration created by this SP.

I bit confused on the permission assignment, Could you please help me to address this issue ? 

I need all the reservation created by all users so that we can import resource using the user creds.

**Comment by DCMattyG** [comment]
Hi @Veljen, can you tell me more about the Terraform module you are using?

The only one I'm aware of that was build for the Azure IPAM solution is this one:

https://registry.terraform.io/providers/XtratusCloud/azureipam/latest/docs

Can you share the link to the one you mentioned above?
